### PR TITLE
move cloud team long duration jobs to multi clusters

### PR DESCRIPTION
**Pull request by sunzhaohua2**
@huali9 PTAL

### CODE REVIEW

**Comment by sunzhaohua2** [comment]
assign @liangxia

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml [hunk 1]
```diff
@@ -42954,6 +42454,94 @@ periodics:
...
+    org: openshift
+    repo: openshift-tests-private
+  labels:
+    ci-operator.openshift.io/cloud: azure4
+    ci-operator.openshift.io/cloud-cluster-profile: azure-qe
+    ci-operator.openshift.io/variant: multi-nightly
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-disc-fullypriv-amd-mixarch-f28-longduration-cloud
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:42474]: The `reporter_config` for slack notification are lost

Comment (sunzhaohua2) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:42474]: Thanks, have added.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 2]
```diff
@@ -186,6 +186,26 @@ tests:
...
+  cron: 7 11 4 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKERS: "1"
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      E2E_RUN_TAGS: '@heterogeneous and @aws-ipi and @disconnected and @network-ovnkubernetes
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:199]: `@network-xxx` is no longer needed.  Refer to https://github.com/openshift/release/pull/47019 fore more details.

Comment (sunzhaohua2) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml]: Done, removed @network-ovnkubernetes

**Comment by liangxia** [comment]
There is a typo in the commit message, `milti`.  Could you help fix it ? Once the PR is merged, we can't change the commit history.

**Pull request merged**